cmd/boilerplate.go

```diff
@@ -138,6 +166,30 @@ local config = require("` + pluginName + `.config")
 	}
 	defer mainLuaFile.Close()
 	mainLuaFile.WriteString(`local config = require("` + pluginName + `.config")`)
+
+	healthLua := filepath.Join(luaPath, "health.lua")
+	healthLuaFile, healthErr := os.Create(healthLua)
+	if healthErr != nil {
+		log.Fatal(healthErr)
+	}
+	defer healthLuaFile.Close()
+	healthLuaFile.WriteString(`local function checkhealth()
+   if vim.fn.has('nvim-0.5') then
```

Comment:
Actually, vim.fn is not available in older neovim versions

For the version check specifically, you should rely on vimscript `has('nvim-0.5')` for example.

(TJ DeVries @tjdevries Aug 31 2020 15:55
yes, vim.fn is not available until 0.5, source https://gitter.im/neovim/neovim?at=5f4d00a449148b41c9852114 )

Reply:
I wonder if at this point it isn't just more convenient to write it in Vimscript directly? To avoid that the lua function just becomes a wrapper around Vim grammar without adding much (and therefore generating confusion)?

Reply:
writing a healthcheck in vimscript is 100% ok, all the more for simple plugins.

While I did write my health() in lua,that was nothing more than an example, and for the kind of checking you (probably) want to provide, vimscript can do the job

Reply:
Taken care of in commit 2694e2ad7841ba73380c5939af7b056344e18f12